sources/editor/Xenko.GameStudio/GameStudioWindow.xaml

```diff
@@ -433,51 +433,51 @@
           <xcad:LayoutPanel Orientation="Horizontal">
             <xcad:LayoutPanel Orientation="Vertical">
               <xcad:LayoutPanel Orientation="Horizontal">
-                <xcad:LayoutAnchorablePane DockWidth="240">
+                <xcad:LayoutAnchorablePane DockWidth="300">
                   <!-- SOLUTION EXPLORER -->
-                  <xcad:LayoutAnchorable ContentId="SolutionExplorer" Title="{Binding Source={xk:Localize Solution explorer, Context=View}}"
-                                         gs:AvalonDockHelper.IsVisible="{Binding Panels.SessionExplorerPanelVisible, Mode=TwoWay,
-                                         Source={x:Static gs:GameStudioViewModel.GameStudio}, FallbackValue={xk:True}}">
+                  <xcad:LayoutAnchorable AutoHideMinWidth="300" ContentId="SolutionExplorer" Title="{Binding Source={xk:Localize Solution explorer, Context=View}}"
+                                           gs:AvalonDockHelper.IsVisible="{Binding Panels.SessionExplorerPanelVisible, Mode=TwoWay,
```

Comment:
Nit: unnecessary whitespace change (+10 more occurrences below).

Reply:
I will fix it, i think was my VS :(

Reply:
Done!